src/configs/beta.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* @ts-ignore */
+import { baseConfig, chooseUrl } from "./configs/base";
+
+const config = {
+    ...baseConfig,
+    isBeta: true,
```

Comment:
The beta config is the mainnet config with `isBeta: true`, so we should import the mainnet config and inherit the values just like we do with the `baseConfig`